src/shared/pages/team/editTeam/__tests__/editTeam.spec.tsx

```diff
@@ -77,6 +107,48 @@ describe('when the editTeam component is mounted', () => {
         });
     });
 
+    it('should hide the edit name area if the user does not have the RENAME_TEAM role', async () => {
+        setDefaults();
+
+        hasRole.mockImplementation((role: string) => {
+            if (role === 'RENAME_TEAM') {
+                return false;
+            } else if (role === 'REASSIGN_TEAM_UNIT') {
+                return true;
+            }
+            return false;
+        });
+
+        act(() => {
+            wrapper = renderComponent();
+        });
+
+        await wait(() => {
+            expect(wrapper.container).toMatchSnapshot();
+        });
+    });
+
+    it('should hide the change unit area if the user does not have the REASSIGN_TEAM_UNIT role', async () => {
+        setDefaults();
+
+        hasRole.mockImplementation((role: string) => {
+            if (role === 'RENAME_TEAM') {
+                return true;
+            } else if (role === 'REASSIGN_TEAM_UNIT') {
+                return false;
+            }
+            return false;
```

Comment:
Could potentially be simplified like:

```suggestion
            if (role === 'RENAME_TEAM') {
                return true;
            } 
            return false;
```